src/components/app-content-switcher/AppContentSwitcher.tsx

```diff
@@ -3,7 +3,7 @@ import { TypographyProps } from '@mui/material/Typography'
 import { Typography } from '@mui/material'
 import { SxProps } from '@mui/system'
 import Tooltip from '@mui/material/Tooltip'
-import Switch from '@mui/material/Switch'
+import AppSwitch from '~/design-system/components/switch/Switch'
```

Comment:
I think we should name all imports without the prefix `App`. As I can see, the `Switch` component has the name `AppSwitch`, but all components inside the design-system folder have names without `App`. So, I think it should be changed in a separate task. What do you think?

Reply:
Maybe, but I am not the created of the component
<img width="271" alt="Screenshot 2024-12-15 at 15 01 59" src="https://github.com/user-attachments/assets/8e16adb5-b116-4b65-86e7-687beba6ef55" />
+ we have got a lot of other components which prefix is app so it wouldn't be cool just to go around the whole project and remove one word:)

Reply:
I'm talking about only `design-system` folder here, not about `components` folder. You're importing `Switch` component from `design-system` folder, not from `components`, and all components there (except Switch) have naming without `App`.

Reply:
Fair enough, fixed.